docker-compose.yml

```diff
@@ -94,6 +94,7 @@ services:
       - ~/.ivy2:/root/.ivy2:cached
       - ~/.m2:/root/.m2:cached
       - ~/.sbt:/root/.sbt:cached
+      - ~/.cache/coursier:/root/.cache/coursier:cached
```

Comment:
Newest versions of SBT now user couriser and all of the download dependencies are stored in this folder.